src/components/network-chart/NetworkChartRow.js

```diff
@@ -69,22 +83,21 @@ const PHASE_OPACITIES = PROPERTIES_IN_ORDER.reduce(
   {}
 );
 
-type NetworkChartRowProps = {
-  +index: number,
-  +marker: Marker,
-  // Pass the payload in as well, since our types can't express a Marker with
-  // a specific payload.
-  +networkPayload: NetworkPayload,
-  +timeRange: StartEndRange,
-  +width: CssPixels,
-  +threadIndex: ThreadIndex,
-};
-
-type State = {|
-  pageX: CssPixels,
-  pageY: CssPixels,
-  hovered: ?boolean,
-|};
+function NetworkPhase({ name, previousName, value, duration, ...style }) {
```

Comment:
note: this is _not_ a react component, but just a function. I didn't want to make it a react component because I didn't want to type it explicitely. But if you prefer this way I can do it.